### ISSUE TITLE
Default rendering shows /header.plain.html and /footer.plain.html

### ISSUE THREAD

**Issue by kptdobe** [adobe/helix-pages#24]
Easy to reproduce, load any repo with helix-pages (like https://helix-templates-content-kptdobe.project-helix.page/)

If no header.md and/or footer.md is found in the repo, the ESI includes fails and the path to the request is returned.

Solution / workaround is being tracked as part of https://github.com/adobe/helix-publish/issues/83

**Comment by trieloff** [comment]
You might be able to hide the error text like this:

```css
footer {
  color: gray; /* same as background */
}
footer * {
  color: black;
}
```

**Comment by kptdobe** [comment]
With helix-cli 4.2.2, it is even worst: `{ "code": "DlAfvgKJwLGfddNqWGXE5OnI7jOPyPRl", "error": "Response is not valid 'message/http'." }` is shown.

> You might be able to hide the error text like this:

After the first read, I thought it was a joke but thinking about it, I think now you are serious :)
This would be a really ugly temporarily dirty weird hack but I assume I'll have to live with it for now 👍

**Comment by kptdobe** [comment]
This is back (reverted by https://github.com/adobe/helix-pages/pull/36)........
cc @davidnuescheler.

**Comment by davidnuescheler** [comment]
i think we need this fixed properly in the ESI handling (see #10), i now understood that we are looking for a temporary workaround on this. i would propose a remove of the content of header and footer in case they are empty in javascript, as this seems to impact the DOM the least and would be forward compatible.

**Comment by kptdobe** [comment]
Temp fix merged: https://github.com/adobe/helix-pages/pull/48
